core/src/main/java/org/eclipse/dash/licenses/http/HttpClientService.java

```diff
@@ -67,7 +68,16 @@ public int post(String url, String contentType, String payload, Consumer<String>
 					Thread.sleep(1000 * tries);
 					continue;
 				}
-				
+				if (response.statusCode() == 429 && tries++ < maxRatelimitRequestsPerExecure) {
+					long saftyMargin = 100l;
+					logger.info("HTTP response 429 (ratelimit)");
+					Optional<String> oResetTime = response.headers().firstValue("x-ratelimit-reset");
```

Comment:
Nit: I would inline this line with the next to avoid the quasi Hungarian Notation. IMHO, it would read better and isn't a violation of the Law of Demeter.